types/web3/providers.d.ts

```diff
@@ -1,11 +1,11 @@
-interface JsonRPCRequest {
+export interface JsonRPCRequest {
```

Comment:
Why do you export this?

Reply:
Exporting the interfaces `JsonRPCRequest` and `JsonRPCResponse` to have the typings for `payload` as well as `response` for this method --

```
web3.currentProvider.send(payload, (err, response) => {}))
```